File: shared/src/androidMain/kotlin/com/habitrpg/shared/habitica/models/invitations/PartyInvite.java
package com.habitrpg.shared.habitica.models.invitations;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class PartyInvite extends RealmObject {

    @PrimaryKey
    private String id;
    private String name;
    private String inviter;

    /**
     * @return The id
     */
    public String getId() {
        return id;
    }

    /**
     * @param id The id
     */
    public void setId(String id) {
        this.id = id;
    }

    /**
     * @return The name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name The name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * @return The inviter
     */
    public String getInviter() {
        return inviter;
    }

    /**
     * @param inviter The inviter
     */
    public void setInviter(String inviter) {
        this.inviter = inviter;
    }
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/models/invitations/PartyInvite.kt
package com.habitrpg.android.habitica.models.invitations

import io.realm.RealmObject
import io.realm.annotations.RealmClass

@RealmClass(embedded = true)
open class PartyInvite : RealmObject(), GenericInvitation {
    override var id: String? = null
    override var name: String? = null
    override var inviter: String? = null
}
